smbclient.c

```diff
@@ -314,7 +302,11 @@ zend_module_entry smbclient_module_entry =
 	, PHP_RSHUTDOWN(smbclient)	/* request_shutdown_func */
 	, PHP_MINFO(smbclient)		/* info_func             */
 	, PHP_SMBCLIENT_VERSION		/* version               */
-	, STANDARD_MODULE_PROPERTIES
+	, PHP_MODULE_GLOBALS(smbclient)
+	, PHP_GINIT(smbclient)		/* globals ctor */
+	, NULL						/* globals dtor */
+	, NULL						/* post_deactivate_func */
```

Comment:
Probably because my ide is configured with tab = 4 spaces (so probably better to align comments with only spaces)